elasticsearch/Chart.yaml

```diff
@@ -4,7 +4,7 @@ maintainers:
 - email: helm-charts@elastic.co
   name: Elastic
 name: elasticsearch
-version: 7.1.1
+version: 7.1.2
```

Comment:
Could you remove this version bump please? More information about this can be found in the [contributing guide](https://github.com/elastic/helm-charts/blob/master/CONTRIBUTING.md#version-bumps) and is also a checklist item in the pull request template